Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -0,0 +1,254 @@
+    var errorDescription: String? {
+        get { return errorView.errorLabel.text }
+        set { errorView.errorLabel.text = newValue }
+    }
+
+    let accountManager = BRBAccountManager()
+
+    private var isLoading: Bool = false {
+        didSet {
+            if isLoading {
+                activityIndicator.startAnimating()
+                loginButton.setTitle(nil, for: .normal)
+            } else {
+                activityIndicator.stopAnimating()
+                loginButton.setTitle("LOG IN", for: .normal)
+            }
+
+            stackView.isUserInteractionEnabled = !isLoading
+            loginButton.isEnabled = !isLoading
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        accountManager.delegate = self
+
+        loginStackView = UIStackView(frame: .zero)
+        loginStackView.axis = .vertical
+        loginStackView.distribution = .fill
+        loginStackView.spacing = 10
+        stackView.insertArrangedSubview(loginStackView, at: 2)
+        stackView.setCustomSpacing(20, after: subtitleLabel)
+        stackView.setCustomSpacing(40, after: loginStackView)
+        loginStackView.snp.makeConstraints { make in
+            make.width.equalToSuperview()
+        }
+
+        setUpPrivacyStatementButton()
+        setUpErrorView()
+        setUpNetidViews()
+        setUpPasswordViews()
+        setUpSkipButton()
+    }
+
+    private func setUpPrivacyStatementButton() {
+        privacyStatementButton = UIButton(type: .system)
+        privacyStatementButton.setTitle("Privacy Statement", for: .normal)
+        privacyStatementButton.setTitleColor(.white, for: .normal)
+        privacyStatementButton.titleLabel?.font = .preferredFont(forTextStyle: .subheadline)
+        privacyStatementButton.setTitleColor(.black, for: .highlighted)
+        privacyStatementButton.addTarget(self,
+                                         action: #selector(privacyStatementButtonPressed(_:)),
+                                         for: .touchUpInside)
+        loginStackView.addArrangedSubview(privacyStatementButton)
+    }
+
+    @objc private func privacyStatementButtonPressed(_ sender: UIButton) {
+        let privacyStatementViewController = BRBPrivacyStatementViewController()
+        self.present(privacyStatementViewController, animated: true, completion: nil)
+    }
+
+    private func setUpErrorView() {
+        errorView = BRBLoginErrorView(frame: .zero)
+        errorView.isCollapsed = true
+        loginStackView.addArrangedSubview(errorView)
+    }
+
+    private func setUpNetidViews() {
+        netidPrompt = UILabel(frame: .zero)
+        netidPrompt.text = "NetID"
+        netidPrompt.textColor = .white
+        netidPrompt.font = .preferredFont(forTextStyle: .headline)
+        loginStackView.addArrangedSubview(netidPrompt)
+
+        netidTextField = UITextField(frame: .zero)
+        netidTextField.textColor = UIColor(white: 225.0 / 255.0, alpha: 1.0)
```

Comment:
do we use this `UIColor` anywhere else in the app? if so, lets make add it as extension of `UIColor`

Reply:
`.white`?

Reply:
Ah, I see this is not the color white.